include/aspect/utilities.h

```diff
@@ -130,6 +130,11 @@ namespace aspect
                                            const double semi_major_axis_a,
                                            const double eccentricity);
 
+      /**
+       * This enum lists available coordinate systems that
+       * can used for the function variables. Allowed values
+       * are 'cartesian', 'spherical', and 'depth'.
```

Comment:
I think it would be nice to actually document what these correspond to. For example, for the spherical coordinate system, say that you are using r,phi,theta where theta is the latitude (or is it the co-latitude)? And how is the 'depth' system related to the others?